src/fhirConfig.ts

```diff
@@ -23,8 +23,21 @@ export interface OAuthStrategy {
     revokeUrl?: string;
 }
 
+/**
+ * http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#using-well-known
+ */
+export interface SmartStrategy extends OAuthStrategy {
+    capabilities: string[];
+    scopesSupported?: string[];
+    responseTypesSupported?: string[];
```

Comment:
fyi, Doesn't appear to be able to be a type: https://tools.ietf.org/html/rfc6749#section-3.1.1

Reply:
The SMART on fhirClient defined `response_types_supported` as an array of string. 
https://github.com/smart-on-fhir/client-js/blob/ce794fb/src/types.d.ts#L745

Also the example from the [spec](http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#sample-response) shows an array of string. 

`"response_types_supported": ["code", "code id_token", "id_token", "refresh_token"],`

Based on the definition in the [link](https://tools.ietf.org/html/rfc6749#section-3.1.1) provided, `response_types` are all string. There's an option to [define new response types](https://tools.ietf.org/html/rfc6749#section-8.4), but those new response types are string also.